curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/62dabe2ef403a12d5d295273.md

```diff
@@ -32,12 +32,23 @@ assert(
 );
 ```
 
-The link's text should be `cat photos`. You have either omitted the text or have a typo.
+The link's text should be cat photos. You have either omitted the text or have a typo.
```

Comment:
Please, don't remove backticks around "cat photos". They are important both for clarity and for translations. 
Also, I don't think this is relevant for step 12. Campers should not change that p element in this step.

Reply:
Yes sorry about that I will add the back ticks.